src/components/Board.re

```diff
@@ -5,11 +5,11 @@ module Card = {
   let make = (~value, _children) => {
     ...component,
     render: (_self) =>
-      if (value == 0) {
```

Comment:
`==` is fine here because it's done against ints. But in general `==` uses physical equality (aka mostly structural) and is disrecommended. I've turned it into a pattern matching to side-step the problem completely